loadtest/control/actions.go

```diff
@@ -143,34 +155,40 @@ func LeaveChannel(u user.User) UserActionResponse {
 func JoinTeam(u user.User) UserActionResponse {
 	userStore := u.Store()
 	userId := userStore.Id()
-	teams, err := userStore.Teams()
+	teamIds, err := u.GetAllTeams(0, 100)
```

Comment:
What was the issue with userStore.Teams() ?

Reply:
The main issue was that `memstore.Teams()` returned the teams the user is in as consequence of a previous call to `GetTeamsForUser` (`memstore.SetTeams` is affected by the same problem `SetUsers` was).

Also, when joining a team from the webapp you go to the `/select_team` page which does an API call to GetAllTeams so I believe it's correct to make an initial call to fetch all teams as part of the action.